.rubocop.yml

```diff
@@ -446,3 +446,9 @@ RSpec/ExampleLength:
   Enabled: true
   Max: 15
   StyleGuide: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ExampleLength
+
+RSpec/MultipleExpectations:
+  Description: Checks if examples contain too many `expect` calls.
```

Comment:
Seeing as we have the `aggregate_failures` ability, I am worried that this might cause people to end up writing very long-running specs bc they limit even heavy lifting specs to a single assertion. I'm wondering if we might be better off without this cop. Thoughts?

Reply:
The cop is going to activate after 8 `expect()` statements which I think it'd be a reasonable number to have in a single spec. If not we can raise it? I understsand the point about speed and I agree but there's also to consider the reason why this cop even exists: too many responsibilities tested in the same test.

What do you think?

Reply:
OH the threshold is 8, well that is much better, I assumed it was 1 🤦‍♀️